controller/src/test/java/io/pravega/controller/fault/ControllerClusterListenerTest.java

```diff
@@ -52,13 +62,9 @@
 
 
     @Before
-    public void setup() {
+    public void setup() throws Exception {
```

Comment:
In general, I prefer to fail explicitly rather than let unit failed with an exception. It makes intent explicit.